packages/js/product-editor/src/blocks/product-fields/product-details-section-description/edit.tsx

```diff
@@ -29,14 +29,13 @@ import { useEntityId } from '@wordpress/core-data';
  * Internal dependencies
  */
 import { ProductEditorSettings } from '../../../components';
-import { ProductTemplate } from '../../../components/editor';
 import { BlockFill } from '../../../components/block-slot-fill';
 import { useValidations } from '../../../contexts/validation-context';
 import {
 	WPError,
 	getProductErrorMessage,
 } from '../../../utils/get-product-error-message';
-import { ProductEditorBlockEditProps } from '../../../types';
+import { ProductEditorBlockEditProps, ProductTemplate } from '../../../types';
```

Comment:
simple suggestion:

```suggestion
import type { ProductEditorBlockEditProps, ProductTemplate } from '../../../types';
```

Reply:
TIL! Thanks for sharing this. There's likely a lot of places where we could switch to doing this.

Reply:
indeed